modules/core/test/v2/unit/coins/xlm.ts

```diff
@@ -112,23 +112,62 @@ describe('XLM:', function() {
     basecoin.isValidStellarUsername('Foo@bar.baz').should.equal(false); // only lowercase letters are allowed
   });
 
-  it('Should be able to explain an XLM transaction', co(function *() {
+  it('Should explain an XLM transaction', co(function *() {
     const signedExplanation = yield basecoin.explainTransaction({ txBase64: 'AAAAAMDHAbd3O7B2auR1e+EH/LRKe8IcQBOF+XP2lOxWi1PfAAAB9AAEvJEAAAABAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAABAAAAB1RFU1RJTkcAAAAAAQAAAAAAAAABAAAAALgEl4p84728zfXtl/JdOsx3QbI97mcybqcXdfgdv54zAAAAAAAAAAEqBfIAAAAAAAAAAAFWi1PfAAAAQDoqo7juOBZawMlk8znIbYqSKemjgmINosp/P4+0SFGo/xJy1YgD6YEc65aWuyBxucFFBXCSlAxP2Z7nPMyjewM=' });
     signedExplanation.outputAmount.should.equal('5000000000');
+    signedExplanation.outputAmounts.should.have.property('txlm');
```

Comment:
Can save a line and do `.property('txlm', '5000000000')` (there are a few times this pattern is repeated). See https://shouldjs.github.io/#assertion-property

Reply:
nice, fixed